WireUI/Sources/WireFolderPickerUI/Resources/Localizable.xcstrings

```diff
@@ -150,7 +150,7 @@
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "How to add a conversation to a folder"
+            "value" : "Learn more"
```

Comment:
Hmm I see that all untranslated entries still have the old string, do we need to update those too or are new strings not used and the fallback is the base string?

Reply:
@johnxnguyen Oh right. That sucks. I'll take a look but perhaps I need to do this manually.

Reply:
@johnxnguyen I've removed the translations as this seems to be the state original state we have before sending localization to Crowdin. However, lets check what happens after crowdin does it's thing